ext/storage_sources/azure_store/azure_connection.cpp

```diff
@@ -91,7 +91,22 @@ azure_connection::delete_object(const std::string &object_key) const
 }
 
 int
-azure_connection::get_object(const std::string &path) const
+azure_connection::read_object(
+  const std::string &object_key, off_t offset, size_t len, void *&buf) const
 {
+    auto blob_client = _azure_client.GetBlockBlobClient(_object_prefix + object_key);
+    auto blob_properties = blob_client.GetProperties().Value;
+    std::vector<uint8_t> downloaded_blob(blob_properties.BlobSize);
+
+    // Downloads the content of the blob to the vector downloaded_blob
+    blob_client.DownloadTo(downloaded_blob.data(), downloaded_blob.size());
+    if (offset + len > downloaded_blob.size()) {
+        std::cerr << "Reading past end of file!" << std::endl;
+        return -1;
+    }
+
+    std::string content =
+      std::string(downloaded_blob.begin() + offset, downloaded_blob.begin() + offset + len);
```

Comment:
could you add a comment briefly explaining what line 109 does?

Reply:
resolved https://github.com/wiredtiger/wiredtiger/pull/8738/commits/81b571a5438a27a5d569b49d4602a7878b0c46f5 let me know if it needs more detail!